### ISSUE TITLE
Clip from unrelated streamer plays during BRB/Hype

### ISSUE THREAD

**Issue by TheTaraStark** [OELSHAMI-ttv/TTV-ClipShoutOut#2]
Every so often, the BRB/Hype player plays this specific clip from an unrelated streamer (the three dudes on a couch, CTR is me) (my attached video has no sound, but when the clip plays there is sound):

https://github.com/OELSHAMI-ttv/TTV-ClipShoutOut/assets/84986402/7171720f-9bc5-4e83-9609-2a7679666443

Multiple different ppl have reported this issue in comments on the Youtube tutorial for this project. It seems we're all seeing the same people, and from my experience it is always this exact clip. It _usually_ happens as the first clip when starting BRB/Hype, but not always. No other BRB/Hype player I've seen encounters this issue (Sidekick, Twitch Guru, Sirbotis, etc), so my guess is it's a parsing issue in the code rather than the API itself, as has been previously suggested. I did some digging and nothing stuck out to me, but my coding skills are also amateur at best.

Hoping this helps? Would love to start using this again, but the random clip popping in makes this unusable for unattended BRBs.

Thanks for your work on this!

**Comment by OELSHAMI-ttv** [comment]
Yeah, I've seen but I've never been able to replicate it or catch it live myself. If for no other reason to get the MP4 name of that clip so I can at least write a filter. I will say though you might want to try the updated code as that may no longer encounter the issue https://github.com/OELSHAMI-ttv/TTV-ClipShoutOut/tree/V3/ShoutOut/OBS and try ShoutoutrepeatReleaseOBSBetterJS.htm

What I really need is the dang channel/Slug that is coming from so I can better understand why the API is leaking that back to me, or if there are a set of clips that can trigger this. Its one of these issues I've been chasing for awhile but if I can't recreate makes it harder to figure out where it is coming from.

**Comment by TheTaraStark** [comment]
Don't know if this is helpful, but I caught an error pop up in the OBS log when the clip played:

`16:51:41.696: [obs-browser: 'Tara Clips (local)'] Error: Uncaught TypeError: Cannot read properties of undefined (reading 'length') (http://absolute/C:/Users/Tara/iCloudDrive/Stream%20Resources/ShoutoutBRB%20clips/ShoutOut/OBS/ShoutoutrepeatReleaseOBS.htm:139)`

**Comment by OELSHAMI-ttv** [comment]
Closing issue because change in how I have to pull the clip using a different interface should prevent this cross loading of a clip.
